### ISSUE TITLE
[Bug] fail-on-eol-found not failing action

### ISSUE THREAD

**Issue by IamGreut** [xeol-io/xeol-action#15]
XEOL is succesfully identifying EOL versions, but not failing the GitHub action when --fail-on-eol-found is true

The following output is shown in the `Run xeol-io/xeol-action@v1.0.7` step

```
Run xeol-io/xeol-action@v1.0.7
/usr/bin/chmod +x /home/runner/work/_temp/648ec04b-47db-44d5-a28e-52b3b9[15](https://github.com/mapbox/thing-for-you/actions/runs/6943289188/job/18888033942?pr=131#step:9:16)74ef
/home/runner/work/_temp/648ec04b-47db-44d5-a28e-52b3b91574ef -b /home/runner/work/_temp/648ec04b-47db-44d5-a28e-52b3b91574ef_xeol v0.9.3
[info] checking github for release tag='v0.9.3' 
[info] fetching release script for tag='v0.9.3' 
[info] checking github for release tag='v0.9.3' 
[info] using release tag='v0.9.3' version='0.9.3' os='linux' arch='amd64' 
[info] installed /home/runner/work/_temp/648ec04b-47db-44d5-a28e-52b3b91574ef_xeol/xeol 
xeol output...
  Executing: xeol -o table --fail-on-eol-found localbuild/testimage:latest
NAME    VERSION  EOL         DAYS EOL  TYPE   
Ubuntu  18.04    2023-05-31  174       os      
node    14.15.4  2023-04-30  205       binary
```

Is it expected the GH Action will "Pass" and continue forward even though EOL has been found?

**Comment by noqcks** [comment]
> Is it expected the GH Action will "Pass" and continue forward even though EOL has been found?

Nope, it should fail here! Good find, I'll work on a fix right now.

**Comment by noqcks** [comment]
I was able to reproduce on v1.0.7 of the action. It seems like this was an issue related to setting argument values that was fixed in PR https://github.com/xeol-io/xeol/pull/229 and included in version v0.9.10 of Xeol. 

Version v1.0.8 of the action should include the fix and we should now fail a CI pipelines when --fail-on-eol-found is passed.

Thanks again for the report!

**Comment by IamGreut** [comment]
Hey @noqcks! Thanks for such a hasty response and fix!
I confirm this is now failing upond finding EOL 😄
